### PR TITLE
Fix Table formatting issue with html

### PR DESCRIPTION
**Pull request by enyia21**
#185454049

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -833,32 +833,56 @@ SubmissionField is an interface that represents a mapping between your applicati
...
+<div>
+<b>Outside of a subflow:</b>
+<br>new SingleField("exampleInputName", "exampleInputValue", null)<br><br>
+</div>
+<div>
+<b>Inside of a subflow:</b>
+<br>new SingleField("exampleInputName", "exampleInputValue", 1)<br>
+</div>
+</td>
+<tr>
```
Comment (bseeger) [inline README.md:858]: I think you are missing a </tr> tag above this.

File: README.md [hunk 2]
```diff
@@ -833,32 +833,56 @@ SubmissionField is an interface that represents a mapping between your applicati
...
-Inside of a
-subflow**: <br> new CheckboxField("exampleInputName", List.of("exampleValueOne", "exampleValueTwo"),
-
-1) |
-                                                                                                | DatabaseField | For fields from database columns. Represents a mapping between database fields
-   such as `submittedAt`, `submissionId`, etc and their values. Does not include an iteration. |
-   Params: `String` database column name, `String` database field value | new DatabaseField("
-   submittedAt", "exampleSubmittedAtValue")
-   |
+<table>
```
Comment (bseeger) [inline README.md:836]: This renders really nicely.  Does it allow for you to indent the HTML to make it more readable?  Not sure if that's possible or not.

File: README.md (hunk 1 above)
Comment (enyia21) [reply README.md:858]: fixed

**Review by bseeger — APPROVED** [review]
Looks good to me!

**Pull request merged**